docs/source/api/model/transform/conditioning.rst

```diff
@@ -0,0 +1,19 @@
+Model Conditioning
+------------------
+
+.. currentmodule:: pymc.model.transform.conditioning
+.. autosummary::
+   :toctree: generated/
+
+   do
+   observe
+
+Others
+------
+
+.. autosummary::
+   :toctree: generated/
+
+   change_value_transforms
+   remove_value_transforms
+   replace_vars_in_graphs
```

Comment:
This one should not be listed

Reply:
fixed in https://github.com/pymc-devs/pymc/pull/6879/commits/aaa91a8e6d14f6a37f5b6ab5ae95043fff44f25e